src/components/panel/panel.js

```diff
@@ -1204,12 +1204,19 @@ MdPanelService.prototype.open = function(preset, config) {
  * @returns {!Object} The preset configuration object.
  */
 MdPanelService.prototype._getPresetByName = function(preset) {
-  if (!this._presets[preset]) {
+  var $injector = this._$injector;
+  var v = this._presets[preset];
```

Comment:
What does `v` stand for? Can you please give this a more meaningful name?

Reply:
It's an arbitrary value because I didn't know what it was for.

Maybe the preset -> presetName and v -> preset?

Reply:
That sounds good. It is much more clear.